.clang-format

```diff
@@ -1 +1,4 @@
 BasedOnStyle: Google
+PointerAlignment: Left
+ReferenceAlignment: Left
+DerivePointerAlignment: false
```

Comment:
I got tired of clang-format going back and forth on things between versions, and I think this'll help. I had to reformat the whole code base because apparently it was not being consistent about where the `&` was.